docs/eeprom-tool_json.schema

```diff
@@ -0,0 +1,95 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://revpi.kunbus.com/revpi.schema.json",
+    "title": "RevPi HAT EEPROM JSON",
+    "description": "The JSON format used in the revpi hat eeprom tool for additional configuration",
+    "type": "object",
+    "required": [ "version", "vstr", "pstr", "pid", "prev", "pver", "dtstr", "gpiobanks" ],
+    "properties": {
+        "version": {
+            "description": "RevPi HAT EEPROM format version",
+            "type": "integer",
+            "minimum": 0,
+            "maximum": 65535
+        },
+        "vstr": {
+            "description": "Vendor string",
+            "type": "string",
+            "maxLength": 255
+        },
+        "pstr": {
+            "description": "Product name",
+            "type": "string",
+            "maxLength": 255
+        },
+        "pid": {
+            "description": "Product identifier",
+            "type": "integer",
+            "minimum": 0,
+            "maximum": 65535
+        },
+        "prev": {
+            "description": "Product revision",
+            "type": "integer",
+            "minimum": 0,
+            "maximum": 65535
+        },
+        "pver": {
+            "description": "Customer visible product version",
+            "type": "integer",
+            "minimum": 0,
+            "maximum": 65535
+        },
+        "dtstr": {
+            "description": "DeviceTree overlay name for this device",
+            "type": "string"
+        },
+        "gpiobanks": {
+            "description": "GPIOBank configuration (only bank 0 is supported at the moment",
```

Comment:
A parenthesis is missing at the end of the text ")"

Reply:
added